README.md

```diff
@@ -95,6 +95,14 @@ source venv/bin/activate
 pip install -e .
 ```
 
+### Running in docker
+
+To give a try to Copanier quickly, you can use docker:
+
+```bash
+sudo docker build -t copanier:local .
+sudo docker run --name copanier --rm -p 2244:2244 copanier:local
+
```

Comment:
It seems we're missing the ending quotes here.

Reply:
You're right, my bad. It's fixed.